store/view/articles.js

```diff
@@ -0,0 +1,168 @@
+import knitService from '~/services/knitService'
+import settings from '~/store-settings/articles'
+import commonHelper from '~/helpers/common'
+import storeHelper from '~/helpers/store'
+import knitLogger from '~/config/logger'
+import _ from 'lodash'
+
+// Opera module
+const { compounds, callers, composer, opuses } = storeHelper.opera
+
+// Mutation types
+const types = {
+  CHANGE_CURRENT_ARTICLE_CODE: 'CHANGE_CURRENT_ARTICLE_CODE',
+  CHANGE_CURRENT_CATEGORY_CODE: 'CHANGE_CURRENT_CATEGORY_CODE',
+  LOAD_ARTICLES: 'LOAD_ARTICLES',
+  ADD_CODES: 'ADD_CODES'
+}
+
+// Mutation functions
+const changeCurrCategory = storeHelper.createMutationFn(types.CHANGE_CURRENT_CATEGORY_CODE, 'currentCategoryCode')
+const changeCurrArticle = storeHelper.createMutationFn(types.CHANGE_CURRENT_ARTICLE_CODE, 'currArticleCode')
+const loading = storeHelper.createMutationFn(types.LOAD_ARTICLES, 'loading')
+const codes = storeHelper.createMutationFn(types.ADD_CODES)
+
+// Module getters
+export const getters = {
+  loading: state => state.loading,
+  mainpage: state => state.mainpage,
+  mainlist: state => state.mainlist,
+  all: state => state.all,
+  currArticleCode: state => state.currArticleCode,
+  currentCategoryCode: state => state.currentCategoryCode,
+  categoryCodeList: state => state.currentCategoryCode === 'wszystkie' ? state.mainlist : state.byCategory[state.currentCategoryCode],
+  byCategory: state => state.byCategory
+}
+
+// Module state
+export const state = () => ({
+  totalItems: 0,
+  mainpage: [],
+  mainlist: [],
```

Comment:
`main list` and `main page` are two words, so I think we should change that to `mainPage` and `mainList`